### PR TITLE
fix: Excluding enddate from request

### PR DESCRIPTION
**Pull request by RasmusGraabaek**
Currently the enddate is included in the aggregated measure data request. 
After this pull request, the enddate is excluded

### CODE REVIEW

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs [hunk 1]
```diff
@@ -431,6 +431,86 @@ public RequestCalculationResultQueriesTests(DatabricksSqlStatementApiFixture fix
...
+            .Select(p => p.Quantity.ToString(CultureInfo.InvariantCulture))
+            .ToArray()
+            .Should()
+            .Equal(FirstQuantity)
+            ;
+    }
+
+    [Theory]
+    [InlineAutoMoqData]
+    public async Task GetAsync_RequestFromGridOperatorStartAndEndDataAreEqual_ReturnsResult(
```
Comment (MadsDue) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs:481]:
```suggestion
    public async Task GetAsync_RequestFromGridOperatorStartAndEndDataAreEqual_ReturnsEmptyResult(
```

**Review by ebbeknudsen — APPROVED** [review]

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs [hunk 2]
```diff
@@ -385,7 +385,7 @@ public RequestCalculationResultQueriesTests(DatabricksSqlStatementApiFixture fix
 
     [Theory]
     [InlineAutoMoqData]
-    public async Task GetAsyncRequestFromGridOperatorTotalProductionThirdCorrectionSettlement_ReturnsResult(
+    public async Task GetAsync_RequestFromGridOperatorTotalProductionThirdCorrectionSettlement_ReturnsResult(
```
Comment (dstenroejl) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs:388]: We have decided that our test names must follow a pattern like `<MemberName>_When<Something>_<IsOrDoes>` so I have added "When": 
```suggestion
    public async Task GetAsync_WhenRequestFromGridOperatorTotalProductionThirdCorrectionSettlement_ReturnsResult(
```

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs [hunk 3]
```diff
@@ -431,6 +431,86 @@ public RequestCalculationResultQueriesTests(DatabricksSqlStatementApiFixture fix
             ;
     }
 
+    [Theory]
+    [InlineAutoMoqData]
+    public async Task GetAsync_RequestFromGridOperatorForOneDay_ReturnsResult(
```
Comment (dstenroejl) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs:436]:
```suggestion
    public async Task GetAsync_WhenRequestFromGridOperatorForOneDay_ReturnsResult(
```

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs [hunk 4]
```diff
@@ -431,6 +431,86 @@ public RequestCalculationResultQueriesTests(DatabricksSqlStatementApiFixture fix
...
+            startOfPeriod: startOfPeriodFilter,
+            endOfPeriod: endOfPeriodFilter);
+
+        var sut = new RequestCalculationResultQueries(sqlStatementClient, deltaTableOptions, requestCalculationResultQueriesLoggerMock.Object);
+
+        // Act
+        var actual = await sut.GetAsync(request);
+
+        // Assert
+        actual.Should().NotBeNull();
```
Comment (dstenroejl) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs:466]: Maybe add use of AssertionScope(?)

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs [hunk 5]
```diff
@@ -431,6 +431,86 @@ public RequestCalculationResultQueriesTests(DatabricksSqlStatementApiFixture fix
...
+            .Select(p => p.Quantity.ToString(CultureInfo.InvariantCulture))
+            .ToArray()
+            .Should()
+            .Equal(FirstQuantity)
+            ;
+    }
+
+    [Theory]
+    [InlineAutoMoqData]
+    public async Task GetAsync_RequestFromGridOperatorStartAndEndDataAreEqual_ReturnsNoResult(
```
Comment (dstenroejl) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs:481]:
```suggestion
    public async Task GetAsync_WhenRequestFromGridOperatorStartAndEndDataAreEqual_ReturnsNoResult(
```

**Review by dstenroejl — COMMENTED** [review]
Add a few comments

**Review by dstenroejl — APPROVED** [review]

**Review by johevemi — APPROVED** [review]

**Pull request merged**